tests/console/yast2_ntpclient.pm

```diff
@@ -34,15 +37,15 @@ sub run {
     assert_screen 'yast2_ntp-client_synchronize_without_daemon';
 
     # check Runtime Configuration Policy
-    send_key 'alt-r';
+    wait_screen_change { send_key 'alt-r'; };
     send_key 'up';
     assert_screen 'yast2_ntp-client_runtime_config_up';
     send_key 'down';
-    send_key 'ret';
+    wait_screen_change { send_key 'ret'; };
```

Comment:
shouldn't it be actually a wait_screen_change guarding the previous 'down' key as well?